.gitignore

```diff
@@ -10,3 +10,4 @@ test/integration/targets/truffle/build
 test/integration/targets/web3-1.0.0/types/web3-contracts
 test/integration/targets/ethers/types/ethers-contracts
 test/package-test/types
+.DS_Store
```

Comment:
this should go to your global git ignore. No need to add it here.

Reply:
42b716f